lib/src/components/NxNexusPageHeader/NxNexusPageHeader.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import classnames from 'classnames';
+
+import { HeaderLinkProps, Props, propTypes } from './types';
+
+export { Props };
+
+import './NxNexusPageHeader.scss';
+
+const logoImg = require('../../assets/img/logo_nexus_generic.svg');
```

Comment:
This hard-codes the hexagon icon.  We need it to be specified by the caller

Reply:
> This hard-codes the hexagon icon. We need it to be specified by the caller

Alright, how's this: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/49a4ece75a8f2a439b71d84e6bd5523d75858ac5